actions/search.py

```diff
@@ -52,9 +52,11 @@ def run(self, instance, query, count):
                 "Failed to connect to Splunk Instance {} with error {}".format(splunk_config, err)
             )
 
-        result = self.service.jobs.oneshot(query, params={"output_mode": "json", "count": count})
-        reader = results.ResultsReader(result)
+
```

Comment:
Flake doesn't like the extra newline and fails the CI. Removing it should fix it.

> /home/runner/work/stackstorm-splunk/stackstorm-splunk/pack/actions/search.py:56:9: E303 too many blank lines (2)

Reply:
fixed!